src/pages/documentation/adobe-journey-optimizer/iam/public-classes.md

```diff
@@ -108,13 +84,97 @@ For a full guide on how to use `handleJavascriptMessage`, read [Call native code
 
 <TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
 
+iOS
+
+<Tabs query="platform=ios&function=handle-javascript-message"/>
+
+## Android Interface - Message
+
+The `Message` interface contains the definition of an in-app message and provides a framework to track message interactions via Experience Edge events.
+
+`InternalMessage` objects implementing this interface are created by the AEPMessaging extension, and passed as the `message` parameter in `MessagingDelegate` protocol methods.
+
+## Public functions
+
+### show
+
+Signals to the UIService that the message should be shown.
+
+If `autoTrack` is true, calling this method will result in an "decisioning.propositionDisplay" Edge Event being dispatched.
+
+Android
```

Comment:
Since we're doing the tab without a tab, I'd just mark all these Androids as H4s.

Reply:
should the `iOS` ones be made as h4's as well?

Reply:
actually, i think i missed the tabsblock headers for the android section. i'll add them in to make it uniform with iOS.

Reply:
Ahh ok - yeah, if you have the TabBlock, there's no need to make them H4s.